pypck/pck_commands.py

```diff
@@ -991,11 +1029,66 @@ def activate_scene_relay(
         :return:  The PCK command (without address header) as text
         :rtype:   str
         """
+        return PckGenerator._activate_or_store_scene_relay(
+            scene_id, relay_ports, store=False
+        )
+
+    @staticmethod
+    def store_scene_relay(
+        scene_id: int, relay_ports: Sequence[lcn_defs.RelayPort] = ()
+    ) -> str:
+        """Store the current relay states in the given scene.
+
+        :param    int                scene_id:       Scene id 0..9
+        :param    list(RelayPort)    relay_ports:    Relay ports to store
+                                                     as list
+        :return:  The PCK command (without address header) as text
+        :rtype:   str
+        """
+        return PckGenerator._activate_or_store_scene_relay(
+            scene_id, relay_ports, store=True
+        )
+
+    @staticmethod
+    def _activate_or_store_scene_relay(
+        scene_id: int,
+        relay_ports: Sequence[lcn_defs.RelayPort] = (),
+        store: bool = False,
+    ) -> str:
         if (scene_id < 0) or (scene_id > 9):
             raise ValueError("Wrong scene_id.")
         if not relay_ports:
             raise ValueError("relay_port list is empty.")
         relays_mask = ["0"] * 8
         for port in relay_ports:
             relays_mask[port.value] = "1"
-        return "SZA0{:03d}{:s}".format(scene_id, "".join(relays_mask))
+        if store:
+            action = "S"
+        else:
+            action = "A"
+        return "SZ{:s}0{:03d}{:s}".format(action, scene_id, "".join(relays_mask))
+
+    @staticmethod
+    def beep(sound: lcn_defs.BeepSound, count: int) -> str:
+        """Make count number of beep sounds.
+
+        :param    BeepSound sound:  Beep sound style
+        :param    int       count:  Number of beeps (1..15)
+        :return:  The PCK command (without address header) as text
+        :rtype:   str
+        """
+        if (count < 1) or (count > 15):
+            raise ValueError("Wrong number of beeps.")
+        return "PI{:s}{:02d}".format(sound, count)
```

Comment:
Not necessarily wrong, but for decimal numbers we should pad to three digits unless otherwise expected by PCK syntax (`{:03d}` for count format).